pkg/ddl/backfilling_dist_scheduler.go

```diff
@@ -93,14 +93,20 @@ func (sch *BackfillingSchedulerExt) OnNextSubtasksBatch(
 		return nil, err
 	}
 	logger.Info("on next subtasks batch")
+	pdCli := sch.d.store.(kv.StorageWithPD).GetPDClient()
+	p, l, err := pdCli.GetTS(ctx)
+	if err != nil {
+		return nil, err
+	}
+	ingestTS := oracle.ComposeTS(p, l)
```

Comment:
so all subtasks have the same ts initially?

Reply:
Oh, in order to find duplicate key in different subtasks, we should not use 1 TS for them all.